packages/modal/src/Modal.tsx

```diff
@@ -136,6 +61,8 @@ Modal.propTypes = {
   shouldClose: PropTypes.func,
   className: PropTypes.string,
   setOpen: PropTypes.func,
+  darkMode: PropTypes.bool,
```

Comment:
not related to this PR specifically but what's the status of propTypes? We have them on some components and not others

Reply:
I believe we still need them for those who aren't using TS. We should add to those that are missing.